tests/std/tests/P0980R1_constexpr_strings/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\concepts_matrix.lst

```

Comment:
Can we de-conceptify the test, so we can get EDG coverage earlier? From a very quick glance, it looks like the only concepts dependency is `ranges::equal`. Can we use a lightweight `EqualRanges` function template that calls `std::equal` with begin/end/begin/end, marked as `// TRANSITION, EDG concepts support`?

Reply:
🤞 I added the function. 🤞